telegram/_bot.py

```diff
@@ -9154,6 +9157,81 @@ async def get_star_transactions(
             bot=self,
         )
 
+    async def send_paid_media(
+        self,
+        chat_id: Union[str, int],
+        star_count: int,
+        media: Sequence["InputPaidMedia"],
+        caption: Optional[str] = None,
+        parse_mode: ODVInput[str] = DEFAULT_NONE,
+        caption_entities: Optional[Sequence["MessageEntity"]] = None,
+        show_caption_above_media: Optional[bool] = None,
+        disable_notification: ODVInput[bool] = DEFAULT_NONE,
+        protect_content: ODVInput[bool] = DEFAULT_NONE,
+        reply_parameters: Optional["ReplyParameters"] = None,
+        reply_markup: Optional[ReplyMarkup] = None,
+        *,
+        read_timeout: ODVInput[float] = DEFAULT_NONE,
```

Comment:
do we want to add the field `allow_sending_without_reply` for new methods? I think yes because we have that in `Defaults`?

Reply:
good point. I think for consistency it would be good to add both.